website/blog/2019-10-28-react-native-doctor.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Announcing React Native Doctor
+author: Lucas Bento
+authorTitle: React Native Community
+authorURL: https://twitter.com/lbentosilva
+authorImageURL: https://avatars3.githubusercontent.com/u/6207220?s=460&v=4
+authorTwitter: lbentosilva
+category: announcements
+---
+
+After a ton of work, the React Native Community is excited to launch `react-native doctor`, a new brand-new CLI command to help you out on getting started with React Native and automatically fixing errors with your installation.
+
+Check it out in action:
+
+[VIDEO WITH DOCTOR]
+
+## How it works
+
+The command currently supports most of the softwares and libraries that React Native relies on, such as CocoaPods, Xcode and Android SDK, the way it works is that it will list the issues that your current installation has and, if you choose to, it will try to automatically fix them, if a specific run wasn't able to fix then it will print out a message and a helpful link for you to fix it manually.
```

Comment:
```suggestion
The `doctor` command currently supports most of the software and libraries that React Native relies on, such as CocoaPods, Xcode and Android SDK. With `doctor` we'll find issues with your development environment and give you the option to automatically fix them. If `doctor` is not able to fix an issue, it will display a message and a helpful link explaining how to fix it manually.
```